server/utils/assessments/utils.ts

```diff
@@ -478,32 +478,32 @@ const acctAlertsFromAssessment = (assessment: Assessment) =>
 export {
```

Comment:
I wonder if we could put this in as an eslint rule, so VS code automatically fixes this. I do sometimes do it manually, but I often forget

Reply:
I've added a rule now